exporter/kafkaexporter/README.md

```diff
@@ -24,8 +24,9 @@ The following settings can be optionally configured:
 - `brokers` (default = localhost:9092): The list of kafka brokers.
 - `resolve_canonical_bootstrap_servers_only` (default = false): Whether to resolve then reverse-lookup broker IPs during startup.
 - `client_id` (default = "sarama"): The client ID to configure the Sarama Kafka client with. The client ID will be used for all produce requests.
-- `topic` (default = otlp_spans for traces, otlp_metrics for metrics, otlp_logs for logs): The name of the kafka topic to export to.
-- `topic_from_attribute` (default = ""): Specify the resource attribute whose value should be used as the message's topic. This option, when set, will take precedence over the default topic. If `topic_from_attribute` is not set, the message's topic will be set to the value of the configuration option `topic` instead. 
+- `topic` (default = otlp_spans for traces, otlp_metrics for metrics, otlp_logs for logs): The name of the default kafka topic to export to. When used in conjunction with `topic_from_attribute` or `topic_from_context`, this is the default/fallback topic.
+- `topic_from_attribute` (default = ""): Specify the resource attribute whose value should be used as the message's topic. This option, when set, will take precedence over the default topic and `topic_from_context`. If `topic_from_attribute` is not set, the message's topic will be set to the value of the configuration option `topic` instead. 
+- `topic_from_context` (default = ""): Specify the context key with a string value that should be used as the message's topic. This option, when set, will take precedence over the default topic. If `topic_from_context` is not set to a string value, the message's topic will be set to the value of the configuration option `topic` instead. 
```

Comment:
How is this configured in let's saay OTLP receiver? Can a user configure it?

Reply:
I've updated the readme and removed this line from when I was trying to use a basic string as the type. That's no longer the case.  The expectation is that users would interact with the context using the exported functions in a custom component, not via configuration.

Reply:
For it to be interacted with via configuration, you'd have to adopt the prefix-driven values like the attributesreceiver does, right? Things like `metadata:` for gRPC metadata, etc.